share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
> Allow customisation of this sort makes software worse. Look through our issue tracker and tell me honestly if all users were using binaries we would have more issues rather than less.

The "number of issues" is not the only relevant consideration in judging what Homebrew should do.